stylesheets/_mixin.responsive-utilities.scss

```diff
@@ -91,3 +91,20 @@
         @include sr-hide;
     }
 }
+
+@mixin word-wrap() {
+    -ms-word-break: break-all; // ie11
+    -ms-hyphens: auto;
```

Comment:
Do we need all these ms specific prefixes? autoprefixer should handle these.

Reply:
Also, should this be in responsive-utilities as it's not responsive specific? Would be better to follow convention and add a new `_mixin.word-wrap.scss` file.

Reply:
We should maybe think about creating a separate include file to include mixins so we don't have to make a breaking change every time we want to introduce a new mixin...

Separate ticket for that though

If this is currently not a breaking I'd leave it in responsive and we can look at introducing a mixin includes file as a separate breaking change